src/components/DataCollector/createEvent.js

```diff
@@ -28,6 +28,9 @@ export default () => {
     mergeDevice: createMerger(content, "device"),
     mergeEnvironment: createMerger(content, "environment"),
     mergePlaceContext: createMerger(content, "placeContext"),
+    setTimestamp: timestamp => {
```

Comment:
I recommend we stick with this syntax where possible: `setTimestamp(timestamp) {`